### PR TITLE
Add command line flag to enable Key Manager API

### PR DESCRIPTION
**Pull request by joaquim-verges**
First PR that starts introducing the Key Manager API as discussed in #439 

This introduces the boolean command line flag that controls whether or not to enable the new API.

### CODE REVIEW

**Comment by jframe** [comment]
Your build is failing due to the formatting errors in the spotless check. You can run ./gradlew spotlessApply to fix the formatting issue.

File: commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java [hunk 1]
```diff
@@ -65,13 +65,24 @@
       converter = UInt64Converter.class)
   private UInt64 altairForkEpoch;
 
+  @CommandLine.Option(
+      names = {"--enable-key-manager-api"},
+      paramLabel = "<BOOL>",
+      description = "Whether to enable the key manager API to manage key stores (default: ${DEFAULT-VALUE}).",
```
Comment (jframe) [inline commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java:71]: Perhaps simplify this to "Enable the key manager API to manage key stores"

Comment (joaquim-verges) [reply commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java]: updated + fixed formatting

**Review by jframe — APPROVED** [review]
LGTM - Just need to fix formatting again for spotless

**Comment by joaquim-verges** [comment]
Should have ran it after the string change 😅 Done.

**Pull request merged**